roles/arrstack/templates/buildarr/instances/sonarr-instances.yml.j2

```
@@ -0,0 +1,176 @@
+---
+# Sonarr instance configuration in Buildarr.
+
+sonarr:
+  port: 8989
+  protocol: "http"
+  hostname: {{ hostname_sonarr | default('localhost') }}
+  settings:
+    quality:
+      trash_id: "bef99584217af744e404ed44a33af589"  # series
+
+  # Instance-specific configuration.
+  instances:
+
+    # Sonarr instance for grabbing HD/SD TV shows.
+    sonarr-hd:
+      hostname: {{ hostname_sonarr | default('localhost') }}
+      api_key: "{{ arrstack_instance_apikeys['sonarr-hd'] if arrstack_instance_apikeys is defined and 'sonarr-hd' in arrstack_instance_apikeys else 'ansible_api_key_not_found' }}"
```

Putting in an incorrect value when something is not found is an anti-pattern I believe.

Instead, let Ansible raise an error.

```suggestion
      api_key: "{{ arrstack_instance_apikeys['sonarr-hd'] }}"
```